<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MasternodeList</class>
 <widget class="QWidget" name="MasternodeList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="topLayout" stretch="0">
   <property name="margin">
    <number>9</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="frame_Header">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_Header" stretch="1,0,1">
          <item>
           <widget class="QLabel" name="labelOverviewHeaderLeft">
            <property name="minimumSize">
             <size>
              <width>464</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>60</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>20</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>MASTERNODES</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="labelOverviewHeaderRight">
            <property name="minimumSize">
             <size>
              <width>464</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>60</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_BG">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="margin">
         <number>6</number>
        </property>
        <item>
         <widget class="QFrame" name="frame_Content">
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_count_2">
           <property name="text">
            <string>Node Count:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="countLabelDIP3">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QTableWidget" name="tableWidgetMasternodesDIP3">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Service</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Status</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>PoSe Score</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Registered</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Last Paid</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Next Payment</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Payout Address</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Operator Reward</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Collateral Address</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Owner Address</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Voting Address</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
